pkgs/misc/lilypond/default.nix

```diff
@@ -31,6 +31,7 @@ stdenv.mkDerivation rec{
     sed -re 's%("[$]exe" --version .*)([|\\] *$)%\1 | sed -re "s@/nix/store/[a-z0-9]{32}-@@" \2%' \
       -i configure
 
+    export CFLAGS="$CFLAGS -fno-delete-null-pointer-checks"
```

Comment:
Since there is an upstream bug filed for this, can we expect that once upstream fixes it we can remove this line? If so, I think we should add a comment referring to the upstream bug report near that line.

Reply:
done.

Reply:
@jasom please add the comment in the code

Reply:
@FRidh @ryantm sorry, I neglected to add the change to the commit.